mod/01_rating/spec/set/type/metric_value_spec.rb

```diff
@@ -54,6 +54,31 @@
       value_card = Card["#{@metric_value.name}+value"]
       expect(value_card.content).to eq("I'm fine, I'm just not happy.")
     end
+
+    it 'creates metric value with source' do
```

Comment:
does this mean it's currently possible to create a metric value without a source?